src/TNetwork.cpp

```diff
@@ -235,7 +238,8 @@ std::shared_ptr<TClient> TNetwork::Authentication(TConnection&& RawConnection) {
         ClientKick(*Client, fmt::format("Invalid version header: '{}' ({})", std::string(reinterpret_cast<const char*>(Data.data()), Data.size()), Data.size()));
         return nullptr;
     }
-    if (!TCPSend(*Client, StringToVector("S"))) {
+
+    if (!TCPSend(*Client, StringToVector("A"))) { //changed to A for Accepted version
```

Comment:
This breaks all clients, doesn't it? Is this a breaking change? Clients will expect `S`, but get `A`, unless I'm misunderstanding this change

Reply:
nope, this is just a byte to make the client continue it doesn't check it just wait for any signal

Reply:
So it doesnt care if it's A or S? What is the idea then with making it send A

Reply:
It needs to wait for the server to finish authentication obviously otherwise there would be no point of continuing it just waits for anything back other than a disconnect